src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,53 @@
+import { Link } from "@canonical/react-components";
+import { useParams } from "react-router-dom";
+
+export type EntityDetailsRoute = {
```

Comment:
You can import this type from the `Routes` component.

```ts
import type { EntityDetailsRoute } from "components/Routes/Routes";
```

Reply:
Nice, done